Makefile

```diff
@@ -1,4 +1,4 @@
-CONTAINER_NAME=nats-request-multi-js_request-multi_1
+CONTAINER_NAME=nats-multi-request-js-request-multi-1
```

Comment:
I'm not 100% sure what the naming conventions here would be, but if we're reversing the naming from `request-multi` to `multi-request`, shouldn't we do it in both parts of this name?